js/components/App.js

```diff
@@ -24,6 +38,8 @@ export default Relay.createContainer(App, {
             node {
               id,
               name,
+              enabled,
+              ${ToggleWidgetEnableMutation.getFragment('widget')},
```

Comment:
Weird. I don't really understand that error. Maybe @wincent or someone from FB can explain?